doc/md/versioned/execution.mdx

```diff
@@ -0,0 +1,156 @@
+---
+id: execution
+slug: /versioned/execution
+title: Migration Execution
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Atlas has its own migration execution engine, that works with the Atlas migration file format, e.g. migration files
+generated by [`atlas migrate diff`](/versioned/diff).
+
+```shell
+atlas migrate
+```
+
+### Arguments
+`atlas migrate apply` accepts one positional integer argument to specify how many pending migration files to run.
+* `atlas migrate apply` run all pending migrations
+* `atlas migrate apply 0` run all pending migrations
```

Comment:
```suggestion
```
Feels unnecessary

Reply:
I wanted to make clear 0 stands for all and not for nothing.